README.md

```diff
@@ -1,6 +1,7 @@
 ## LMAX Disruptor
 
 ![Java CI with Gradle](https://github.com/LMAX-Exchange/disruptor/workflows/Java%20CI%20with%20Gradle/badge.svg)
+[![CodeQL](https://github.com/fillreject/disruptor/workflows/CodeQL/badge.svg)](https://github.com/fillreject/disruptor/actions?query=workflow%3ACodeQL)
```

Comment:
Not sure we can take this, linked to the @fillreject fork.
Might be better to have the PR without changes to this badge and do the change direct on the repo after

Reply:
I removed the commits with this change.